torchci/scripts/checkRockset.mjs

```diff
@@ -18,45 +18,53 @@ function printDiff(first, second) {
   });
 }
 
-async function checkQuery(client, queryName, version) {
+async function checkQuery(client, workspace, queryName, version) {
   console.log(
-    `Checking that commons.${queryName}:${version} is matches your local checkout`
-  );
-  const qLambda = await client.queryLambdas.getQueryLambdaVersion(
-    "commons",
-    queryName,
-    version
+    `Checking that ${workspace}.${queryName}:${version} is matches your local checkout`
   );
+  let qLambda;
+  if (version === "latest") {
```

Comment:
similarly, I would suggest that we only support checking by exact version hash, not tags, since tags can be re-pointed to other things.